experiments/AtmosLES/convective_bl_model.jl

```diff
@@ -189,20 +192,25 @@ function init_convective_bl!(problem, bl, state, aux, localgeo, t)
     state.ρ = ρ
     state.ρu = SVector(ρu, ρv, ρw)
     state.ρe = ρe_tot
-    state.moisture.ρq_tot = ρ * q_tot
+    if !(bl.moisture isa DryModel)
+        state.moisture.ρq_tot = ρ * q_tot
+    end
 
     if z <= FT(400) # Add random perturbations to bottom 400m of model
         state.ρe += rand() * ρe_tot / 100
     end
     init_state_prognostic!(bl.turbconv, bl, state, aux, localgeo, t)
 end
 
-function surface_temperature_variation(state, t)
+function surface_temperature_variation(state, t, moisture_model)
```

Comment:
In the end we are not using these arguments anymore for `surface_temperature_variation`. You can copy what we are doing in the stable bl simulation.